src/services/schema-form.js

```diff
@@ -307,6 +307,8 @@ angular.module('schemaForm').provider('schemaForm',
 
           angular.forEach(schema.properties, function(value, key) {
             var path = options.path.slice();
+            path.push(key);
```

Comment:
I should probably add a comment here later, but basically this needs to be here before the def because `path` is used to create the def.